src/containers/ScheduleTable/ScheduleTable.tsx

```diff
@@ -90,7 +90,7 @@ const ScheduleTable = <T extends Pair,>({ schedule }: ScheduleWrapperProps<T>) =
     <GridContainer>
       {currentDayColumn ? <CurrentDayContainer start={currentDayColumn}/> : null}
       <ScheduleHeader />
-      {generateScheduleRows(generateScheduleMatrix(weekSchedule))}
+      {generateScheduleRows(generateScheduleMatrix(weekSchedule, data?.data.currentLesson!))}
```

Comment:
Avoid using non-null assertion operator. It breaks the concept of type checks itself. `data` here clearly is undefined at some point of time, so there's no way this param would be 100% non-null. Although it'll probably work here fine, but potential harm is worse than benefits of it.
Better do it this way: pass `data?.data.currentLesson` as is, but in `generateScheduleMatrix` make this parameter optional with default value, like `-1` for instance, which is expected inside and treated as a fallback.
This way all possible types are respected and seen along the way, and don't bring confusion.

Reply:
> Avoid using non-null assertion operator. It breaks the concept of type checks itself. `data` here clearly is undefined at some point of time, so there's no way this param would be 100% non-null. Although it'll probably work here fine, but potential harm is worse than benefits of it. Better do it this way: pass `data?.data.currentLesson` as is, but in `generateScheduleMatrix` make this parameter optional with default value, like `-1` for instance, which is expected inside and treated as a fallback. This way all possible types are respected and seen along the way, and don't bring confusion.

Ok. As for detecting the current pair - `activePair !== -1` is the first check. I will provide 0 as the default value, so if no current lesson is provided will do a proper check.

Reply:
> Just make sure 0 isn't a first pair. Otherwise this could result in visual "jumps" of active pair during data loading. Though I can see I left there placeholder data with 0's, so it must be like that already, or works fine with that. Idk, just check it to be sure. If so, maybe change placeholder data default values to -1 as well.

As I can see, from the `/current/time` endpoint, the current lesson returns "as is", so the first pair is 1. It should work fine.

Reply:
Alrighty then.